crates/red_knot_python_semantic/src/types.rs

```diff
@@ -1232,6 +1244,9 @@ impl<'db> KnownClass {
             | Self::Set
             | Self::Dict => module.name() == "builtins",
             Self::GenericAlias | Self::ModuleType | Self::FunctionType => module.name() == "types",
+            Self::Sized | Self::Hashable => {
```

Comment:
basically all of the "interesting" protocols in `typing` are deprecated aliases at this point to members of `collections.abc`.